pipelines/HSC/SingleFrame.yaml

```diff
@@ -1,9 +1,12 @@
 description: >-
   Single-frame pipeline for the case in which
   no templates exist, specialized for HSC
+  Unlike ap_pipe/SingleFrame.yaml, this pipeline is not standalone.
 
 imports:
   - location: $AP_PIPE_DIR/pipelines/HSC/SingleFrame.yaml
-    include:
-      - apPipeSingleFrame
-      - analyzeCalibrationMetrics
+    exclude:
+      # Run in prep_butler or Preprocessing.yaml
+      - getRegionTimeFromVisit
+      - mpSkyEphemerisQuery
+      - loadDiaCatalogs
```

Comment:
It is true we don't want to run `loadDiaCatalogs` in SingleFrame, but I don't actually see that in the imports so I think the exclusion is unnecessary.

Reply:
Yes, it's not there. This was intended as "future-proofing" against changes to `SingleFrame` (which I think is part of what caused this mess in the first place). We want to include any changes to upstream `SingleFrame`, _except_ changes that incorporate more than preprocessing.

And of course I tested that excluding a nonexistent task is legal. 🙂